### DISCUSSION TITLE
modify Webhook URL with imageFile variable

### DISCUSSION THREAD

**Discussion by JoeyJoe6** [djdd87/SynoAI#161] [Q&A]
Im attempting to utilize a webhook under Notifiers. The URL is a API call that I need to add the &image name to the end. 

https://api.callmebot.com/signal/send.php?phone=[xxx]&apikey=[xxx]&image=image.jpg

I believe I have found the variable string ImageField that SynoAI utilizes when "SendImage": "true" is noted. I have attempted to utilize this string in my API call with no luck. Could anyone suggest how I could accomplish this? I am able to issue this if I utilize curl in a script, but not sure how I could execute this script with in the SynoAI Notifiers section.

**Comment by djdd87** [comment]
The name will be different every time, but there's no way of accessing this currently to place it in the URL. I could probably add a property to just override it so it's always just "image.jpg" as the filename is pretty irrelevant at the point it's posted out.

**Reply by JoeyJoe6** [reply]
keeping different file names would be best so you can review manually later if needed. 

Im playing with the notion of monitoring the STDout and scraping the filename and path into variables. ill see how that goes. I see in STDout "Saving image to 'Captures/cam2-alerts/cam2-alerts_2022_10_21_17_51_55_358_2.jpeg'  Im a newbe so ill have to research :-), or "Image saved to" but i should be able to generate some variables to use in a script. 

But ideally adding in a notifier to utilize json and passing the filename and path would be ideal :-)
